src/main/java/twilightforest/integration/TFTinkerConstructIntegration.java

```diff
@@ -80,10 +81,10 @@ public class TFTinkerConstructIntegration {
 
     public static final class MaterialID {
 
-        public static final int FieryMetal = 42;
-        public static final int Knightmetal = 43;
-        public static final int NagaScale = 44;
-        public static final int Steeleaf = 45;
+        public static final int FieryMetal = TwilightForestMod.FieryMetal_ID;
```

Comment:
@DrParadox7 did it mean i can set it to a new id if id 42-45 is occupied ?

Reply:
Yes, that's correct